public_suffix_list.dat

```diff
@@ -14077,7 +14077,13 @@ co.pl
 
 // Microsoft Corporation : http://microsoft.com
 // Submitted by Public Suffix List Admin <msftpsladmin@microsoft.com>
+// Managed by Corporate Domains
+// Microsoft Azure : https://home.azure
+cloudapp.azure.com
```

Comment:
Since domains issued on `cloudapp.azure.com` are in the format `<label>.<region>.cloudapp.azure.com` shouldn't the rule be `*.cloudapp.azure.com` so that domains issued to vms in the same region are treated as different?

Reply:
Thank you for calling that out, I have updated it to the wildcard